Dockerfile

```diff
@@ -76,6 +78,11 @@ RUN set -ex \
     python2 python39 \
     # Packages needed for AWT.
     libXext libXrender libXtst libXi libX11-xcb mesa-libgbm libdrm freetype \
+    # Arch specific installs of libsecret and libsecret-devel
+    && if [[ $(uname -m) == "s390x" ]]; then cd /tmp && curl -sSLO https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-0.20.4-2.fc34.s390x.rpm && curl -sSLO https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-devel-0.20.4-2.fc34.s390x.rpm && microdnf install -y --nodocs dnf && dnf install -y -q --nodocs /tmp/libsecret*.rpm && rm -fr /tmp/libsecret*.rpm; fi \
```

Comment:
by keeping this as a single line, it's WAAAAY easier to sed-remove it in downstream :D

Reply:
`dnf` is not available in ubi8-minimal

Reply:
--> by keeping this as a single line, it's WAAAAY easier to sed-remove it in downstream :D

it may be more robust to call external script like `./install-platform-dependencies.sh` and then script can be just a no-op downstream than sed everywhere

Reply:
true, but `microdnf install -y --nodocs dnf && dnf install -y -q ...` will make it available :D 

And yeah, moving this to a separate sh script will be my next iteration. good idea!

Reply:
> --> by keeping this as a single line, it's WAAAAY easier to sed-remove it in downstream :D
> 
> it may be more robust to call external script like `./install-platform-dependencies.sh` and then script can be just a no-op downstream than sed everywhere

nice idea, it can be done with separate pr